### PR TITLE
Prepare the 1.5.0 release.

### LINKED ISSUE #583: Release 1.4.9
This will be a bugfix release for 1.4.8 (#560) immediately followed by a 1.5.0 release to introduce the new functionality / ~semi-public API change in #456.

On the docket:
+ Fix resolve regressions introduced by the 1.4.8. (#580)

### PR DESCRIPTION
**Pull request by jsirois**
Closes #583
Work towards #585

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,27 @@
 Release Notes
 =============
 
+1.4.9
+-----
+
+This is a hotfix release for 1.4.8 that fixes a regression in interpreter setup that could lead to
+resolved distributions failing to build or install.
+
+* Support environment markers during pex activation. (#582)
```
Comment (jsirois) [inline CHANGES.rst]: Hrm... wasn't thinking. This commit should not be in the release.
Thinking now how I want to handle.

Comment (jsirois) [reply CHANGES.rst]: Alright - I guess the only real option is to just do 1.5.0. I'll bump the version and change the descriptions.

Comment (jsirois) [reply CHANGES.rst]: Alright - fixed up. Sorry about this folks!

**Comment by jsirois** [comment]
It does. This was what I was weighing. As noted in the [1.5.0 ticket](https://github.com/pantsbuild/pex/issues/585) we break "API"s without bumping our major version - which is a problem. The key bit here is defining our API formally. Informally, everyone uses the CLI as the API and only Pants uses PEX as a library - so this only breaks Pants - which is fine, I'll fix that on upgrade. That said - we can't be sure Pants is the only API user, thus #584

**Comment by jsirois** [comment]
The only other reasonable option here is to revert 5f1f00f767 and then release 1.4.9. Perhaps I should just do that.

**Comment by jsirois** [comment]
At the cost of thrashing reviewers, I'll close this and proceed with a revert, new 1.4.9 PR, revert-revert, 1.5.0 PR

**Comment by Eric-Arellano** [comment]
I agree reverting 5f1f00f is the best option 👌

**Pull request closed without merging**